src/tests/java17/CollectionTest.java

```diff
@@ -0,0 +1,23 @@
+package java17;
```

Comment:
Same here and for the other classes; please add them in the existing namespace, as we want to use `javaXX` only for features that specifically introduces in a given Java version.